docs/uns-network-key-concepts/network-governance.md

```diff
@@ -15,17 +15,17 @@ tags:
 
 # Understanding governance of uns.network blockchain
 
-The uns.network blockchain is a DPOS kind of blockchain. It count three separates categories of stakeholders, with specific purpose and interest. They are:
-- Organization: which make secure confidential connections on their web site ;
+The uns.network blockchain is a DPOS kind of blockchain. It counts three separates categories of stakeholders, with specific purpose and interest. They are:
+- Organization: which make secures confidential connections on their web site ;
 - Individual: which own UNS credits ;
 - Community actors which work for the network only.
 
-Therefore governance is based on delegates colleges, allowing that all @unikname types to be represented. Colleges are organized separately and indepantely. Each network members can only voted for its own represantes and according to rules specific to its college:
+Therefore, governance is based on delegates colleges, allowing that all @unikname types to be represented. Colleges are organized separately and independently. Each network member can only vote for his own representatives according to rules specific to his college:
 
-- Organization are forced to vote. At each time than a @unikname is created with organization type, it have to choose a representative. It can be him-self or another member of its college. Each @unikname organization give its poll weighted by its subscription to the Unikname Connect solution. By this way, organization's delegates are garants of the adequacy between uns.network technology and Unikname Connect solution. 10 places are reserved for delegates with type organization.
+- Organizations are forced to vote. At each time than a @unikname is created with organization type, it have to choose a representative. It can be himself or another member of its college. Each @unikname organization give its poll weighted by its subscription to the Unikname Connect solution. By this way, organization's delegates are the guarantors of the adequacy between the uns.network technology and Unikname Connect solution. 10 places are reserved for delegates with type organization.
```

Comment:
```suggestion
- Organizations must vote. At each time than a @unikname is created with organization type, it have to choose a representative. It can be himself or another member of its college. Each @unikname organization give its poll weighted by its subscription to the Unikname Connect solution. By this way, organization's delegates are the guarantors of the adequacy between the uns.network technology and Unikname Connect solution. 10 places are reserved for delegates with type organization.
```